src/ttmp32gme/build/file_handler.py

```diff
@@ -370,3 +379,32 @@ def open_browser(host: str, port: int) -> bool:
     except Exception as e:
         logger.error(f"Could not open browser: {e}")
         return False
+
+
+def create_oid_images_zip(library_path: Optional[Path] = None) -> Optional[io.BytesIO]:
```

Comment:
The library path argument is now obsolete. However since ghetto ID cash now gets the library path from the database, we need to move this entire function to DB handler to avoid circular dependencies